pkg/cli/run.go

```diff
@@ -355,3 +368,20 @@ func outputApp(out io.Writer, format string, app *apiv1.App) error {
 	}
 	return err
 }
+
+func toggleHiddenFlags(cmd *cobra.Command, flagsToHide []string, hide bool) {
+	for _, flag := range flagsToHide {
+		cmd.PersistentFlags().Lookup(flag).Hidden = hide
+	}
+}
+
+func setAdvancedHelp(cmd *cobra.Command) {
+	cmd.SetHelpFunc(func(cmd *cobra.Command, args []string) {
+		fmt.Println(cmd.Short + "\n")
+		// toggle advanced flags on before printing flags out in cmd.UsageString
+		toggleHiddenFlags(cmd, []string{"dangerous", "memory", "target-namespace", "secret", "volume", "region", "publish-all",
```

Comment:
Could this be hard coded as a constant variable at the top? I see this list repeated three times.